scripts/monitoring/cron-send-docker-metrics.py

```diff
@@ -8,15 +8,17 @@
 # Adding the ignore because it does not like the naming of the script
 # to be different than the class name
 # pylint: disable=invalid-name,import-error
+# pylint: disable=line-too-long
```

Comment:
Please put reason we are disabling the particular pylint, and if it only for one line, you can put the disable next to the line that requires it.